updater/spec/dependabot/updater_spec.rb

```diff
@@ -2234,6 +2236,29 @@ def expect_update_checker_with_ignored_versions(versions)
 
       updater.run
     end
+
+    it "does not include ignored dependencies in the group PR" do
```

Comment:
I think it would be helpful to also include a test checking the length of `dependency_change.updated_dependencies` to show it only updates 1 dependency, so if it's ignored then there is no PR created.